pyproject.toml

```diff
@@ -61,6 +61,7 @@ dependencies = [
   "numpy",
   "npyscreen",
   "omegaconf",
+  "onnx",
```

Comment:
- I had to install this after updating to `main` in order for the app to run
- then I also had to install `onnxruntime` manually, I guess because `onnx` was installed, otherwise the app did not start up

I did not ask for `onnx`. So I'm not sure the optional dependencies are working as expected.